### DISCUSSION TITLE
Keywords component

### DISCUSSION THREAD

**Discussion by alxhslm** [nunocoracao/blowfish#1109] [Ideas]
Thanks @nunocoracao for creating such a wonderful theme 🙌! I'm using the theme on my personal website and it looks great. 

I'm putting a CV on my about page, and wanted a list of keywords in my skills section. I could have used the badge component, but I couldn't get the spacing right. The spacing between badges (when used to tag articles) is set outside the `badge.html` itself within [`article-meta/basic.html`](https://github.com/nunocoracao/blowfish/blob/ea8344a4bc9b6dfaf915bd1b08565673d65b0de5/layouts/partials/article-meta/basic.html#L83). I couldn't see any easy way to get it to work without refactoring.

After more thought, I decided that  I actually wanted to differentiate these keywords from badges anyway. Keywords are not "alerts" for new or important information, but more of a container, so arguably it makes sense for them to be a different component.

I've implemented a version of this which is now live example on my website:
<img width="287" alt="Screenshot 2023-12-09 at 22 03 46" src="https://github.com/nunocoracao/blowfish/assets/46283223/783fa95a-7ee4-4159-8f24-6eee9d4e9405">

If you agree that this something you want as part of Blowfish, I'm happy to create a PR.

**Comment by nunocoracao** [comment]
Hey @alxhslm are these implemented as shortcodes? 
If so please go ahead with the PR for sure we can work out the details over there - FYI I would ask you to also fill in the respective docs page to make sure people know how to use it

**Comment by alxhslm** [comment]
This was implemented in #1136.
